src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -126,55 +136,63 @@ def test_Tensor_repr():
     machine_representation = repr(obj1)
     assert not "object at 0x" in machine_representation
 
-
+@xfail(raises=TypeError)
```

Comment:
We can make `xfail` to be strict
```suggestion
@xfail(raises=TypeError, strict=True)
```

Reply:
done